### ISSUE TITLE
users: Toggling password auth state (lock/unlock) on root causes a usermod error modal dialog

### ISSUE THREAD

**Issue by garrett** [cockpit-project/cockpit#17009]
### Explain what happens

1. Go to the users page and select the **root** account.
2. Toggle the "Disallow password authentication" option. 
3. See the following dialog:

![image](https://user-images.githubusercontent.com/10246/154080094-310bcacb-e29b-4768-8954-fac4c736014d.png)

### Version of Cockpit

main

### Where is the problem in Cockpit?

Accounts

### Server operating system

Fedora

### Server operating system version

35

### What browsers are you using?

Firefox

**Comment by garrett** [comment]
This is a follow-up to address https://github.com/cockpit-project/cockpit/pull/16908#pullrequestreview-882877569

**Comment by marusak** [comment]
I cannot reproduce this. Can you please create reproducer where you also create an account?

**Comment by garrett** [comment]
> Can you please create reproducer where you also create an account?

It's not when creating an account though?

And it's the root account, on Fedora.

**Comment by marusak** [comment]
> And it's the root account, on Fedora.

I tested this with root on Fedora and works fine. I think this might be that your root does not have password or something. I tried to set up such account but it always worked.

> It's not when creating an account though?

Sure, but we need to reproduce this to fix this and then write test for it. That means we need to know how we need to create account that reproduces this (does it have password, is there some configuration...).
